docs/users-guide/07-dashboards.md

```diff
@@ -1,126 +1,145 @@
+You can amend the URL of a dashboard to automatically enter fullscreen, enable night mode, or auto-refresh the dashboard. Customizing the dashboard's URL allows you to configure the dashboard even when you do not have any input access to the device where the dashboard will be displayed, like scripted screens, for example.
 
 To configure a dashboard using its URL, you can add the following optional keywords: `fullscreen`, `night`, or `refresh`. Here's an example:
 
 `https://metabase.mydomain.com/dash/2#refresh=60&fullscreen&night`
 
 The part that says `refresh=60` sets the dashboard to automatically refresh every 60 seconds, `fullscreen` sets it to fullscreen mode, and `night` sets it to night mode (night mode only works when using fullscreen). Use an `&` in between keywords, and make sure there's a hash `#` after the dashboard's ID number.
 
-There is one important limitation with the `fullscreen` option. For security reasons, many browsers require user interaction to initiate fullscreen. In those browsers, using the `fullscreen` option will enable the fullscreen UI in Metabase, but it won't expand the browser content to fill the screen. To ensure the dashboard occupies the entire screen, either activate fullscreen by clicking the button in the UI or use the `fullscreen` URL option and launch the browser in fullscreen or kiosk mode.
+There is one important limitation with the `fullscreen` option. For security reasons, many browsers require user interaction to initiate fullscreen. In those browsers, using the `fullscreen` option will enable the fullscreen UI in Metabase, but it won't expand the browser content to fill the screen. To ensure the dashboard occupies the entire screen, either activate fullscreen by clicking the button in the UI, or use the `fullscreen` URL option and launch the browser in fullscreen or kiosk mode.
 
-## Tips on creating helpful dashboards
+### Archiving a dashboard
 
-To make a really helpful dashboard, you need to first decide what you want the dashboard to tell about your data. What questions will give you insight into what you want to know? It helps to think of a topic or theme for your dashboard — something like “customer satisfaction,” or “second quarter sales goals,” for example.
+Archiving a dashboard removes the dashboard from searches and collections. Archiving a dashboard does not archive the individual saved questions on it — it just archives the dashboard.
+ 
+To archive a dashboard, click the **pencil** icon to enter edit mode, then click the **...** menu, and select **Archive**.
+
+To view all archived items, click the **menu** icon in the top-right of any collection page. You can unarchive a dashboard by clicking the icon of the box with the upward arrow next to that dashboard.
+
+## Tips on creating helpful dashboards
```

Comment:
Stray thought: seems like this topic could be expanded into a blog post, if it hasn't been already.